### PR TITLE
Simplify naming entity stringify mthod

### LINKED ISSUE #244: Remove assert from naming.entity.stringify
`assert` is dependency on node.js environment

### PR DESCRIPTION
**Pull request by qfox**
fix(naming.entity.stringify): purify method

BREAKING CHANGE: Remove normalization logic from the method

Closes #244

### CODE REVIEW

**Comment by Yeti-or** [comment]
rebase and fix tests

File: packages/naming.entity.stringify/index.js [hunk 1]
```diff
@@ -16,24 +16,15 @@ function stringify(entity, delims) {
 
     let res = entity.block;
 
-    if (entity.elem) {
+    if (entity.elem !== undefined) {
```
Comment (Yeti-or) [inline packages/naming.entity.stringify/index.js]: why not ? `if (!entity.elem)`

File: packages/naming.entity.stringify/index.js [hunk 2]
```diff
@@ -16,24 +16,15 @@ function stringify(entity, delims) {
...
-        const hasModVal = modObj && modObj.hasOwnProperty('val') || entity.hasOwnProperty('modVal');
-        const modVal = modObj && modObj.val || !entity.__isBemEntityName__ && entity.modVal;
-
-        if (modVal || modVal === 0 || !hasModVal) {
-            res += delims.mod.name + modName;
-        }
-
-        if (modVal && modVal !== true) {
-            res += delims.mod.val + modVal;
+    if (entity.mod !== undefined) {
```
Comment (Yeti-or) [inline packages/naming.entity.stringify/index.js]: same

**Review by Yeti-or — COMMENTED** [review]
Do we need to assert for `mod : {name, val}` format?

**Comment by qfox** [comment]
> Do we need to assert for mod : {name, val} format?

No, it's in BemEntityName constructor

File: packages/naming.entity.stringify/index.js (hunk 1 above)
Comment (qfox) [reply packages/naming.entity.stringify/index.js]: Because it's faster and not much larger

**Review by Yeti-or — APPROVED** [review]

**Comment by Yeti-or** [comment]
I think it's problem:
```
      AssertionError: expected 'block_' to equal 'block_mod'

      + expected - actual


      -block_

      +block_mod
```

**Comment by qfox** [comment]
Tests was not changed so I've merged this.

Supported objects:
- `BemEntityName`
- `{ block: string, elem?: string, mod?: string|{ name: string, val?: string|true } }`

Examples:
```js
[
  {block: 'b'},
  {block: 'b', mod: 'value-less-mod'},
  {block: 'b', mod: {name: 'value-less-mod', val: true}},
  {block: 'b', mod: {name: 'usual-mod', val: 'val'}},
]
```

**Pull request merged**